saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -44,6 +48,38 @@ class FulfillmentCancelInput(graphene.InputObjectType):
     restock = graphene.Boolean(description="Whether item lines are restocked.")
 
 
+class FulfillmentClearMeta(ClearMetaBaseMutation):
+    class Meta:
+        description = "Clears meta for fulfillment"
```

Comment:
I would slightly change it:
```suggestion
        description = "Clears metadata for fulfillment."
```
And similarly for the rest of the descriptions.

Reply:
Metadata or meta-data?